js/repl.js

```diff
@@ -47,7 +49,13 @@ function compile() {
 
       var result = e.data;
       if (result.type === 'success') {
-        output.setValue(result.data, -1);
+        var code = result.data;
+        if (isEmpty.test(code) && !isEmpty.test(input.getValue())) {
+          code =
+            '// Your code was completely dead code eliminated.\n' +
```

Comment:
Perhaps "Your code was all dead code and thus eliminated" might be more clear wording?